admin/test/features/support/env.rb

```diff
@@ -56,4 +57,26 @@
   CLIENT = Mongo::Client.new('mongodb://localhost/mtc')
 end
 
+Database = ENV['SQL_DATABASE'] || 'mtc'
+Server = ENV['SQL_SERVER'] || 'localhost'
+Port =  ENV['SQL_PORT'] || 1433
+Admin_User = ENV['SQL_ADMIN_USER'] || 'sa'
+Admin_Password = ENV['SQL_ADMIN_USER_PASSWORD'] || 'Mtc-D3v.5ql_S3rv3r'
```

Comment:
please use ruby variable style ie. admin_password etc Lowercase and spaces replaced with underscores.